codec/src/main/java/io/netty/handler/codec/ConvertibleHeaders.java

```diff
@@ -90,13 +90,6 @@
     List<ConvertedType> getAllAndRemoveAndConvert(UnconvertedType name);
 
     /**
-     * Invokes {@link Headers#entries()} and lazily does a conversion on the results as they are accessed
```

Comment:
@nmittler @Scottmitch yeah I opened an issue for this a week ago or so https://github.com/netty/netty/issues/3655 .